ncp-web-neo/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
accidentally saved file

Reply:
Not _really_ accidentally - but yeah, I should remove that again :D